jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/log/ProcessInstanceLogItemView.java

```diff
@@ -15,121 +15,105 @@
-    public void init() {
-        tooltip(logIcon);
-    }
-
-    @Override
-    public HTMLElement getElement() {
-        return listGroupItem;
-    }
+    private ProcessInstanceLogItemDetailsView workItemView;
 
     @Override
     public ProcessInstanceLogSummary getValue() {
-        return this.logSummary.getModel();
+        return logSummary.getModel();
     }
 
     @Override
     public void setValue(final ProcessInstanceLogSummary model) {
-        String iconClass = "list-view-pf-icon-sm kie-timeline-list-view-pf-icon-sm";
-        this.logSummary.setModel(model);
-        logTime.setTextContent(DateUtils.getPrettyTime(model.getDate()));
+        logSummary.setModel(model);
 
-        String agent = constants.System();
+        setLogTime(model.getDate());
+        setLogIcon(model);
+        setLogInfo(model);
+        setLogType(model);
+        setDetails(model);
+    }
 
-        if (LogUtils.NODE_TYPE_HUMAN_TASK.equals(model.getNodeType()) ||
-                (LogUtils.NODE_TYPE_START.equals(model.getNodeType()) && !model.isCompleted())) {
+    private void setLogIcon(ProcessInstanceLogSummary model) {
+        tooltip(logIcon);
+        String iconClass = "list-view-pf-icon-sm kie-timeline-list-view-pf-icon-sm";
+        if (isHumanTask(model) || (isStartNode(model) && !model.isCompleted())) {
             iconClass += " fa fa-user";
-            agent = constants.Human();
-            tooltip(logIcon);
-            logIcon.setAttribute("data-original-title",
-                                 constants.Human_Task());
+            logIcon.setAttribute("data-original-title", constants.Human_Task());
         } else {
             iconClass += " fa fa-cogs";
-            logIcon.setAttribute("data-original-title",
-                                 constants.System_Task());
+            logIcon.setAttribute("data-original-title", constants.System_Task());
         }
         if (model.isCompleted()) {
             iconClass += " kie-timeline-icon--completed";
-            logCompleted.setTextContent(constants.NodeWasLeft(agent));
-        } else {
-            logCompleted.setTextContent(constants.NodeWasEntered(agent));
         }
         logIcon.setClassName(iconClass);
-        nodeTypeDesc.setTextContent(getLogTitle(model));
+    }
+
+    private void setLogInfo(ProcessInstanceLogSummary model) {
+        String agent = getAgent(model);
+        if (model.isCompleted()) {
+            logInfo.setTextContent(constants.NodeWasLeft(agent));
+        } else {
+            logInfo.setTextContent(constants.NodeWasEntered(agent));
+        }
+    }
 
+    private void setLogType(ProcessInstanceLogSummary model) {
+        String logTitle = model.getNodeType();
+        String name = model.getName();
+        if (isHumanTask(model)) {
+            logTitle = constants.Task_(model.getName());
+        } else if (name != null && name.trim().length() > 0) {
+            logTitle = model.getNodeType() + " '" + name + "' ";
```

Comment:
similar question here

Reply:
the same answer here :)

Reply:
@jsoltes hi, according to [answer 1](https://github.com/kiegroup/jbpm-wb/pull/1255#discussion_r241822777) and [answer 2](https://github.com/kiegroup/jbpm-wb/pull/1255#discussion_r241822894) I think we should remove also this log white space, could you please double check?

Reply:
@jomarko the whitespace is necessary, as it concatenates with logInfo, which does not have whitespace at the beginning.